time/src/weekday.rs

```diff
@@ -66,6 +66,26 @@ impl Weekday {
         }
     }
 
+    /// Get n-th next day.
+    ///
+    /// ```rust
+    /// # use time::Weekday;
+    /// assert_eq!(Weekday::Monday.nth(1), Weekday::Tuesday);
+    /// assert_eq!(Weekday::Sunday.nth(10), Weekday::Wednesday);
+    /// ```
+    pub const fn nth(self, n: u8) -> Self {
+        match (self.number_days_from_monday() + (n % 7)) % 7 {
```

Comment:
```suggestion
        match (self.number_days_from_monday() + n % 7) % 7 {
```

Operator precedence means we don't need parentheses here.